executor/runtime/docker/docker_ssh.go

```diff
@@ -135,21 +123,12 @@ func addContainerSSHDConfigWithData(c runtimeTypes.Container, tw *tar.Writer, cf
 		return err
 	}
 
-	// The format that is used for SSH Users is:
-	// $(unix username):$(app name):$(aws account id):$(task id)
-
+	containerEnv := c.Env()["NETFLIX_ENVIRONMENT"]
 	users := append(cfg.ContainerSSHDUsers, c.AppName())
 	for _, username := range users {
-		lines := []string{}
-		for _, accountID := range accountIDs {
-			lines = append(
-				lines,
-				fmt.Sprintf("%s:%s:%s:%s", username, c.AppName(), accountID, c.TaskID()), // key scoped to username, appname, account ID, and task ID
-				fmt.Sprintf("%s:%s:%s", c.AppName(), accountID, c.TaskID()),              // key has access to any username for this given app in this given account, with this task ID
-				fmt.Sprintf("%s:%s", c.AppName(), accountID),                             // key has access to any username for this given app in this given account
-				c.TaskID(), // key has access to any username on this task ID
-				fmt.Sprintf("%s:%s", username, c.TaskID()), // key has access to this given username on this task ID
-			)
+		lines := []string{
+			fmt.Sprintf("~v3:titus:%s:%s:%s:%s:%s:%s", username, c.AppName(), containerEnv, c.TaskID(), c.JobGroupStack(), c.JobGroupDetail()),
```

Comment:
Kinda same deal here, except this ends up in the file itself.

Can you add a Comment string (not a golang comment, a `fmt.Sprintf("# This line allows Bla, and if any fields are empty, then ssh won't work. The "bless" infrastructure at Netflix is what generates certificates with these principals...`), just a nice to have for the next person looking at the file wondering where these magic strings came from.

Reply:
:+1: Will add a link to the bless docs.